source/documentation/admin-guide/administration-guide.html.md

```diff
@@ -1170,7 +1170,7 @@ This applies to all services, but you should be especially aware of the hazards
 *   Directory Services
 *   DNS
 *   Storage
-    </div>
```

Comment:
Isn't this div closing the div opened at line 1165?  <div class="alert alert-info">